README.md

```diff
@@ -1 +1,21 @@
-# tooling-for-developers
\ No newline at end of file
+# Tooling for developers
+
+## Overview
+
+This repo contains some tools to help developers setting things up in a Windows environment. It contains tools, scripts, extensions and config file that are "nice to have".
+
+> It is still a `WORK IN PROGRESS`
+
+## Getting Started
+
+For the moment, to use it, just clone the repo... From there, you should be "Good to go".
+
+Then execute the scripts. Each folder should contains a readme that explains how to use it.
+
+By the way, We know, it's not elegant but we do trust you will contribute with some PR to improve it :P
```

Comment:
Trust your tools! :) Maybe update the sentence to something more positive? Something like "We're always improving and your contributions are welcome!"?

Reply:
I updated with your suggestions and changed some other stuff in the getting started. I also did some little changes to the blockquote